src/resources/js/tickets-attendees.js

```diff
@@ -1,193 +1,226 @@
+				order_ID: obj.attr( 'data-attendee-id' ),
+				nonce   : Attendees.checkin_nonce
+			};
 
-		var obj = jQuery( this );
+			// add event_ID information if available
+			if ( obj.attr( 'data-event-id' ) ) {
+				params.event_ID = obj.attr( 'data-event-id' );
+			}
 
-		var params = {
-			action  : 'tribe-ticket-checkin-' + obj.attr( 'data-provider' ),
-			provider: obj.attr( 'data-provider' ),
-			order_ID: obj.attr( 'data-attendee-id' ),
-			nonce   : Attendees.checkin_nonce
-		};
-		
-		// add event_ID information if available
-		if ( obj.attr( 'data-event-id' ) ) {
-			params.event_ID = obj.attr( 'data-event-id' );
-		}
+			$.post(
+				ajaxurl,
+				params,
+				function( response ) {
+					if ( response.success ) {
+						obj.parent( 'td' ).parent( 'tr' ).addClass( 'tickets_checked' );
 
-		$.post(
-			ajaxurl,
-			params,
-			function( response ) {
-				if ( response.success ) {
-					obj.parent( 'td' ).parent( 'tr' ).addClass( 'tickets_checked' );
+						$( '#total_checkedin' ).text( parseInt( $( '#total_checkedin' ).text() ) + 1 );
+					}
+				},
+				'json'
+			);
 
-					$( '#total_checkedin' ).text( parseInt( $( '#total_checkedin' ).text() ) + 1 );
-				}
-			},
-			'json'
-		);
+			e.preventDefault();
+		} );
 
-		e.preventDefault();
-	} );
+		$( '.tickets_uncheckin' ).click( function( e ) {
 
+			var obj = jQuery( this );
 
-	$( '.tickets_uncheckin' ).click( function( e ) {
+			var params = {
+				action  : 'tribe-ticket-uncheckin-' + obj.attr( 'data-provider' ),
+				provider: obj.attr( 'data-provider' ),
+				order_ID: obj.attr( 'data-attendee-id' ),
+				nonce   : Attendees.uncheckin_nonce
+			};
 
-		var obj = jQuery( this );
+			// add event_ID information if available
+			if ( obj.attr( 'data-event-id' ) ) {
+				params.event_ID = obj.attr( 'data-event-id' );
+			}
 
-		var params = {
-			action  : 'tribe-ticket-uncheckin-' + obj.attr( 'data-provider' ),
-			provider: obj.attr( 'data-provider' ),
-			order_ID: obj.attr( 'data-attendee-id' ),
-			nonce   : Attendees.uncheckin_nonce
-		};
+			$.post(
+				ajaxurl,
+				params,
+				function( response ) {
+					if ( response.success ) {
+						obj.parent( 'span' ).parent( 'td' ).parent( 'tr' ).removeClass( 'tickets_checked' );
+						$( '#total_checkedin' ).text( parseInt( $( '#total_checkedin' ).text() ) - 1 );
+					}
+				},
+				'json'
+			);
 
-		// add event_ID information if available
-		if ( obj.attr( 'data-event-id' ) ) {
-			params.event_ID = obj.attr( 'data-event-id' );
-		}
+			e.preventDefault();
+		} );
 
-		$.post(
-			ajaxurl,
-			params,
-			function( response ) {
-				if ( response.success ) {
-					obj.parent( 'span' ).parent( 'td' ).parent( 'tr' ).removeClass( 'tickets_checked' );
-					$( '#total_checkedin' ).text( parseInt( $( '#total_checkedin' ).text() ) - 1 );
-				}
-			},
-			'json'
-		);
+		/**
+		 * Handle "move" requests for individual rows.
+		 */
+		$( '.row-actions' ).find( '.move-ticket' ).click( function( event ) {
```

Comment:
Rather than bind click events to x elements resulting in x events being bound, I'd suggest this change:

```
$( '.some-parent-maybe-the-table').on( 'click', '.row-actions .move-ticket', function( event ) {
  // do stuff
} );
```

This will result in 1 event being bound on one element but apply to all `.move-ticket` elements.